### DISCUSSION TITLE
About to Patch my Macan PCM, I have some questions...

### DISCUSSION THREAD

**Discussion by googleg** [Mr-MIBonk/M.I.B._More-Incredible-Bash#376] [Q&A]
Hi all,

I want to perform the hack on my 2017 Macan. It has already Carplay and navigation enabled from the factory, but I am a big fan of the Google ecosystem hence I need Android auto.

Before starting I have a few questions for you guys. I am a bit paranoid and what to make sure I will not do any mistake.

I will go for the "automatic install" as recommended, and copy all the patches to the SD (no selection of individual patches). 

1. I am in the Europe region, does it make any difference?
2. Does the automatic update select the correct version automatically? 
3. Is Android Auto enabled directly by the automatic process or do I need to go into the green menu after patching?
4. Does the automatic process also perform a full backup?
5. As said above, my car already has Carplay enabled, is it a problem (I assume not)?
6. I understand that the BOSE update will NOT be installed, am I correct?
7. Will the automatic update also update the maps (I do not think so)?
8. Can the unit go into sleep mode during the process and if so what do I need to do?
9. It says it is recommended to have an external power connected to the car and not the engine running, is a battery charger connected under the hood good enough?
10. Could a future update at my Porsche dealer disable Android Auto?

Sorry for the long post and thank you for your answers !

Happy new year to all and thanks for making your work available to all.

**Comment by googleg** [comment]
Just a quick question: the automatic process will just patch my FW and not install an updated version correct? So I do not have to worry about the Bose system bug after an update...

**Reply by R1zla1** [reply]
Did you try this ? as im having the same issue (macan 2018), hoping just patching existing firmware will not affect Bose ?

**Reply by demonp82** [reply]
Just patched Macan 2018 with AIO fw. It doesnt harm Bose. Follow the instructions of AIO update. 
You need to download AIO FW, download Gracenote AIO. 
When done, with TUNER+CAR buttons enter the green menu (not that easy). With a help of right turning button you can navigate in green menu and activate functions. Carplay, android, etc and also whatching DVD movies on the go. 
When set, you can update maps with another sd or overwrite it with map updates. 
The whole process took around 1 hour (the maps update is 26gb, this was long). So after updates I recommend going around with your car to charge battery.

**Reply by R1zla1** [reply]
Yeah i didnt want to use AIO , as firmware was older than my cars.

I just patched my firmware and all good   

And yep updated maps straight after :)

**Reply by Buddy220022** [reply]
you said not that easy, how do you bring up the green menu, i tried pressing both buttons the same time and nothing.

**Reply by demonp82** [reply]
> you said not that easy, how do you bring up the green menu, i tried pressing both buttons the same time and nothing.

Press and hold together Tuner and Car buttons. 
It took me around 10 minutes of trials to press it really together. I dont know why, maybe it is only my car. 
Just keep trying!
